File: C#/BirthdayCakeCandles.cs
public static int birthdayCakeCandles(List<int> candles)
{
    int maxLen = 0;
    int maxLenCount = 0;
    foreach(int candle in candles){
        if(candle > maxLen){
            maxLen = candle;
            maxLenCount = 1;
        }else if(candle == maxLen){
            maxLenCount = maxLenCount + 1;
        }
    }
    return maxLenCount;
}

File: Python/BirthdayCakeCandles.py
def birthdayCakeCandles(candles):
    # Write your code here
    maxLen = max(candles)
    maxLenCount = candles.count(maxLen)
    
    return maxLenCount

    # Upper here O(2N)

def birthdayCakeCandles(candles):
    # Write your code here
    maxLen = 0
    maxLenCount = 0
    for i in range(len(candles)):
        if maxLen < candles[i]:
            maxLenCount = 1
            maxLen = candles[i]
        elif maxLen == candles[i]:
            maxLenCount = maxLenCount + 1
    return maxLenCount